### PR TITLE
google-season-of-docs: give details on the budget

### PR DESCRIPTION
**Pull request by maelvls**
cc @wallrj

### CODE REVIEW

File: content/en/docs/contributing/google-season-of-docs/2022/improve-navigation-and-structure/index.md [hunk 1]
```diff
@@ -72,3 +72,15 @@ we will measure a baseline number of clicks required to achieve the task and we
...
+Regarding the amount of $12,000, we assume that it will be enough to fund one experienced technical writer
+part-time (for example, they could work half day from Tuesday to Friday, for a total of 24 days, for 3 months
+at a daily rate of $500).
+
+We will give the "volonteer stipend" to contributors who can show they have one PR within the project
+timeframe (from 1st May to 30th July) in which a re-write of one page or a set of pages. Before
+starting the rewriting, the volonteer will suggest which page they wish to work on either on Slack
+(Kubernetes Slack, channel #cert-manager-dev), or in an issue on GitHub, and make sure by asking the
+team whether it makes sense to rework this page. As long as at least one positive reaction, the
+volonteer can start working. For the stipend to be validated, the PR needs to be reviewed and merged.
```
Comment (wallrj) [inline content/en/docs/contributing/google-season-of-docs/2022/improve-navigation-and-structure/index.md:85]:
```suggestion
We will give the "volunteer stipend" to contributors who can show they have one PR within the project
timeframe (from 1st May to 30th July) in which a re-write of one page or a set of pages. Before
starting the rewriting, the volunteer will suggest which page they wish to work on either on Slack
(Kubernetes Slack, channel #cert-manager-dev), or in an issue on GitHub, and make sure by asking the
team whether it makes sense to rework this page. As long as at least one positive reaction, the
volunteer can start working. For the stipend to be validated, the PR needs to be reviewed and merged.
```

**Review by wallrj — CHANGES_REQUESTED** [review]
Thanks @maelvls 

Just one typo to be fixed then can merge.

File: content/en/docs/contributing/google-season-of-docs/2022/improve-navigation-and-structure/index.md (hunk 1 above)
Comment (maelvls) [reply content/en/docs/contributing/google-season-of-docs/2022/improve-navigation-and-structure/index.md]: Done in 653ede2d.

File: content/en/docs/contributing/google-season-of-docs/2022/improve-navigation-and-structure/index.md [hunk 2]
```diff
@@ -72,3 +72,15 @@ we will measure a baseline number of clicks required to achieve the task and we
...
+Regarding the amount of $12,000, we assume that it will be enough to fund one experienced technical writer
+part-time (for example, they could work half day from Tuesday to Friday, for a total of 24 days, for 3 months
+at a daily rate of $500).
+
+We will give the "volunteer stipend" to contributors who can show they have one PR within the project
+timeframe (from 1st May to 30th July) in which a re-write of one page or a set of pages. Before
+starting the rewriting, the volunteer will suggest which page they wish to work on either on Slack
+(Kubernetes Slack, channel #cert-manager-dev), or in an issue on GitHub, and make sure by asking the
+team whether it makes sense to rework this page. As long as at least one positive reaction, the
+volunteer can start working. For the stipend to be validated, the PR needs to be reviewed and merged.
```
Comment (wallrj) [inline content/en/docs/contributing/google-season-of-docs/2022/improve-navigation-and-structure/index.md:85]:
```suggestion
We will give the "volunteer stipend" to contributors who can show they have one PR within the project
time frame (from 1st May to 30th July) in which a re-write of one page or a set of pages. Before
starting the rewriting, the volunteer will suggest which page they wish to work on either on Slack
(Kubernetes Slack, channel #cert-manager-dev), or in an issue on GitHub, and make sure by asking the
team whether it makes sense to rework this page. As long as at least one positive reaction, the
volunteer can start working. For the stipend to be validated, the PR needs to be reviewed and merged.
```

**Review by wallrj — CHANGES_REQUESTED** [review]
One more spell check error.

```
+++ Running spell check
    content/en/docs/contributing/google-season-of-docs/2022/improve-navigation-and-structure/index.md
       81 | timeframe (from 1st May to 30th July) i 

>> 1 spelling error found in 935 files

```

File: content/en/docs/contributing/google-season-of-docs/2022/improve-navigation-and-structure/index.md (hunk 2 above)
Comment (maelvls) [reply content/en/docs/contributing/google-season-of-docs/2022/improve-navigation-and-structure/index.md]: Thanks. Done in b4a9294.

**Review by wallrj — APPROVED** [review]
/lgtm

**Pull request merged**